run.py

```diff
@@ -144,34 +143,46 @@ def cleanup(log, dicom_directory, output_dir):
     # This prevents the script from coppying in subdirectories to the output.  I don't
     # Think this particular gear does that anyway, though.
     for file_name in src_files:
+
         full_file_name = os.path.join(dicom_directory, file_name)
+
         if os.path.isfile(full_file_name):
-            dest = output_dir / Path(file_name)
+
+            if os.path.splitext(Path(file_name))[1] == '.dcm':
+                if not os.path.exists(dicom_directory / Path('dcms_to_zip')):
+                    os.mkdir(dicom_directory / Path('dcms_to_zip'))
+                dest = dicom_directory / Path('dcms_to_zip') / Path(file_name)
+
+            else:
+                dest = output_dir / Path(file_name)
+
             os.rename(full_file_name, dest)
 
+    col = DICOMCollection.from_dir(dicom_directory / Path('dcms_to_zip'))
+    col.to_zip(output_dir / Path('results.zip'))
```

Comment:
can we preserved the input file basename and use .dicom.zip as extension instead ?
e.g. `CMU-1.svs` -> `CMU-1.dicom.zip`